3rdParty/SDL2/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(SDL_TEST_ENABLED_BY_DEFAULT OFF)
 include(functions/FetchContent_MakeAvailableExcludeFromAll)
 include(FetchContent)
 FetchContent_Declare(SDL2
-    URL https://github.com/libsdl-org/SDL/archive/cb46e1b3f06a08d57b4ccd83127d1ec3139e1c0f.tar.gz
-    URL_HASH MD5=b5539b578ef77f6364f621dc55c244d7
+    URL https://github.com/libsdl-org/SDL/archive/a1d1946dcba6509f0679f507b57e7b228d32e6f8.tar.gz

```

Comment:
Do we also need to update the Android project files?

Reply:
I don't think so, android-project didn't change from 2.24.0 to 2.24.1: https://github.com/libsdl-org/SDL/commits/release-2.24.x/android-project

Reply:
Also, no changes on Android in 2.24.1 release notes

Reply:
There are a few minor fixes and gamepad support additions that where skipped when i upgraded from 2.0.22 to 2.0.23, not that it should hold back this PR